test/unit/uk/gov/hmrc/individualincomedesstub/controller/EmployerControllerSpec.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.individualincomedesstub.controller
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import org.mockito.BDDMockito.given
+import org.scalatest.mock.MockitoSugar
```

Comment:
This `MockitoSugar` is deprecated in `scalatest` version `3.0.1` and hence causes compile time warnings. The same goes for the same import in `EmployerServiceSpec`. Importing `org.scalatest.mock.MockitoSugar` instead, removes the warnings.